tests/datasets/test_substation.py

```diff
@@ -0,0 +1,204 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+import shutil
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import pytest
+import torch
+import torch.nn as nn
+
+from torchgeo.datasets import DatasetNotFoundError, Substation
+
+
+class TestSubstation:
+    @pytest.fixture(
+        params=[
```

Comment:
None of these are actually being used at the moment. We just rerun the same code 10x without changes. See https://docs.pytest.org/en/stable/how-to/parametrize.html for how to use this properly. Also, if it makes the code simpler, you could use `itertools.product` to create the cross product of multiple parameter options.